index.html

```diff
@@ -1,76 +1,639 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
 
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>
+        Food Website
+    </title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.2/css/all.min.css"
+        integrity="sha512-Evv84Mr4kqVGRNSgIGL/F/aIDqQb7xQ2vcrdIwxfjThSH8CSR7PBEakCr51Ck+w+/U6swU2Im1vVX0SVk9ABhg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 
 <body>
-    <div class="first_section">
-        <div class="first_section-heading">
-            <h1 class="first_section-h1"><span>Hi,</span>I am Tithi Saha</h1>
-            <h1 class="first_section-h2">FULL STACK DEVELOPER</h1>
+    <section>
+        <nav>
+            <div class="logo">
+                <img src="./image/logo.png">
+            </div>
+            <ul>
+                <li><a href="#Home">Home</a></li>
+                <li><a href="#About">About</a></li>
+                <li><a href="#Menu">Menu</a></li>
+                <li><a href="#Gallery">Gallery</a></li>
+                <li><a href="#Review">Review</a></li>
+                <li><a href="#Order">Order</a></li>
+            </ul>
+            <div class="icon">
+                <i class="fa-solid fa-magnifying-glass"></i>
+                <i class="fa-solid fa-heart"></i>
+                <i class="fa-solid fa-cart-shopping"></i>
+            </div>
+        </nav>
+        <div class="main">
+            <div class="main_text">
+                <h1>Get Fresh <span>Food</span><br>in a Easy Way</h1>
+            </div>
+            <div class="main_image">
+                <img src="./image/main_img.png">
+            </div>
         </div>
-    </div>
-    <div class="middle_section">
-        <p>WORK, I CAN DO FOR <span>YOU</span></p>
-    </div>
-    <div class="card_section">
-        <div class="card_section-card card1">
-            <div class="card_section-img">
-                <img src="./images/mobile.png" alt="mobile-card">
-            </div>
-            <div class="card_section-text">
-                <p class="card_section-heading">App Development</p>
-                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Recusandae temporibus quaerat ipsum cum,
-                    quia dolore!</p>
-
+        <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Necessitatibus asperiores alias velit assumenda
+            doloremque laudantium, libero iusto sapiente, iste mollitia non ut commodi minus corporis voluptate aliquam!
+            Cumque, dicta?Lorem ipsum dolor sit amet consectetur, adipisicing elit. Corrupti voluptates illum pariatur
+            maxime. Suscipit repellat sed possimus aut sunt hic.
+        </p>
+        <div class="main_btn">
+            <a href="#">Order Now</a>

```

Comment:
```suggestion
            <a href="blank">Order Now</a>
```
use blank instead of using # as it is not considered good practice